### DISCUSSION TITLE
Post-install: Where's the main disk's EFI partition?

### DISCUSSION THREAD

**Discussion by nivlemuk** [valnoxy/t480-oc#61] [Q&A]
Hello. I have followed the Installation instructions and successfully installed Monterey on my T480.

In the post-installation instruction "Install OpenCore to Hard drive", I am unable to locate the "main disk's EFI partition" to copy the EFI folder, as per item 2 on "Open Finder and copy the EFI folder of your USB device to the main disk's EFI partition".

Appreciate if anyone can elaborate further on the instruction please. Thank you.

**Comment by cchewanik** [comment]
You can use this tool to mount your EFI

https://github.com/corpnewt/MountEFI

Instructions are on the web page on how to install it, and run it.

Once its mounted, copy the EFI folder from your USB key to the mounted EFI folder

**Reply by nivlemuk** [reply]
Thank you for your reply. I tried using MountEFI but was still unsure about how to go about it.

I traced back my steps and found the problem: At the Install macOS stage, I did not select "Show all devices" and the "Scheme: GUID Partition Map" did not appear. Repeated step 3 under "Install macOS", fast forward to the "Post-Install (optional)" instructions, and it can now boot macOS without my USB device.
